akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/PredefinedFromStringUnmarshallers.scala

```diff
@@ -78,11 +66,39 @@ trait PredefinedFromStringUnmarshallers {
       }
     }
 
-  val CsvString: Unmarshaller[String, immutable.Seq[String]] =
+  val CsvStringSeq: Unmarshaller[String, immutable.Seq[String]] =
     Unmarshaller.strict[String, immutable.Seq[String]] { string ⇒
       string.split(",").toList
     }
 
+  val CsvByteSeq: Unmarshaller[String, immutable.Seq[Byte]] =
+    CsvStringSeq.map(_.map(byteFromString))
+
+  val CsvShortSeq: Unmarshaller[String, immutable.Seq[Short]] =
+    CsvStringSeq.map(_.map(shortFromString))
+
+  val CsvIntSeq: Unmarshaller[String, immutable.Seq[Int]] =
+    CsvStringSeq.map(_.map(intFromString))
+
+  val CsvLongSeq: Unmarshaller[String, immutable.Seq[Long]] =
+    CsvStringSeq.map(_.map(longFromString))
```

Comment:
> This is not, in fact, a proper CSV unmarshaller; ...

Let's just call it "good enough for typical form data" then. ;) If you need fully fledged csv parsing then you can still plug in your custom unmarshaller, right? I don't think that form fields are a good way to submit nested values, though.